code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -80,6 +80,21 @@
 		qdel(GS)
 		eaten = TRUE
 
+	var/mob/living/carbon/human/PP = locate(/mob/living/carbon/human) in oviewers(1, src)
+	var/list/nommable_parts = list()//a list that'll store the limbs of our victim
+	if(PP)
+		if(istype(PP.dna.species, /datum/species/pod))//is the species of the mob a podperson?
+			for(var/Q in PP.bodyparts) //getting the victim's current body aprts
+				var/obj/item/bodypart/NN = Q
+				if(NN.body_part != CHEST && NN.body_part != HEAD) //getting every limb but the chest & head
+					nommable_parts += NN //adding the limbs we got to the above-mentioned list
+			PP.adjustBruteLoss(10)//nom
+			var/obj/item/bodypart/NB = pick(nommable_parts) //using the above-mentioned list to get a choice of limbs for dismember() to use

```

Comment:
if you're ignoring the chest and head, then you will run out of nommable parts, and thus ```pick()``` will runtime since it doesn't work on empty lists, you need to check the list is non-empty first.

also the plantpeople-eating part of ```eat_plants()``` needs a cooldown, badly.

Reply:
Its fiiiiiine, no cooldown is fine